### ISSUE TITLE
Exception starting lein ritz-nrepl

### ISSUE THREAD

**Issue by lmanolov** [pallet/ritz#46]
I was trying to start lein ritz-nrepl (version 0.4.2) but I got the following exception:

```
$ lein ritz-nrepl
Warning: profile :jpda not found.
Exception in thread "main" clojure.lang.ArityException: Wrong number of args (1) passed to: nrepl$start-jpda-server
    at clojure.lang.AFn.throwArity(AFn.java:437)
    at clojure.lang.AFn.invoke(AFn.java:39)
    at user$eval2716.invoke(NO_SOURCE_FILE:1)
    at clojure.lang.Compiler.eval(Compiler.java:6511)
    at clojure.lang.Compiler.eval(Compiler.java:6501)
    at clojure.lang.Compiler.eval(Compiler.java:6501)
    at clojure.lang.Compiler.eval(Compiler.java:6477)
    at clojure.core$eval.invoke(core.clj:2797)
    at clojure.main$eval_opt.invoke(main.clj:297)
    at clojure.main$initialize.invoke(main.clj:316)
    at clojure.main$null_opt.invoke(main.clj:349)
    at clojure.main$main.doInvoke(main.clj:427)
    at clojure.lang.RestFn.invoke(RestFn.java:421)
    at clojure.lang.Var.invoke(Var.java:419)
    at clojure.lang.AFn.applyToHelper(AFn.java:163)
    at clojure.lang.Var.applyTo(Var.java:532)
    at clojure.main.main(main.java:37)
Subprocess failed
```

I am using Arch Linux with the following JDK:

```
$ java -version
java version "1.7.0_07"
Java(TM) SE Runtime Environment (build 1.7.0_07-b10)
Java HotSpot(TM) 64-Bit Server VM (build 23.3-b01, mixed mode)
```

**Comment by jolby** [comment]
I'm seeing the same issue. 

OS X 10.7
java version "1.6.0_35"
Java(TM) SE Runtime Environment (build 1.6.0_35-b10-428-10M3811)
Java HotSpot(TM) 64-Bit Server VM (build 20.10-b01-428, mixed mode)

**Comment by cola-zero** [comment]
I can reproduce same issue on 0.4.2. But it seems nrepl works on current devel branch.
Here is a log.

```
$ lein ritz-nrepl
Warning: profile :jpda not found.
All namespaces already :aot compiled.
nREPL server started on port 34156
```

Can I ignore this woning? or How I can remove it?

**Comment by hugoduncan** [comment]
Should be fixed in 0.5
